test/TestUserAcceptance.hs

```diff
@@ -570,6 +570,34 @@ testEditingMailHeaders = purebredTmuxSession "user can edit mail headers" $
     sendKeys ", testuser@foo.test\r" (Substring "To: user@to.test, testuser@foo.test")
       >>= assertRegex lastLineIsStatusLine
 
+    step "user can add cc header"
```

Comment:
Cool! Did you find it on your own? I think that's great and would leave it.

Reply:
The config we're using for the user acceptance tests writes the mail out to a file instead of actually sending it. Some tests (e.g. "forwards mail successfully") analyses the "sent" mail by opening it up and asserting that certain sub strings are found. It's very crude but we haven't had a need for something more elaborate.

I wish we would have put the same procedure into the 'sending mail successfully'. It currently only asserts that the mail can be parsed.

Anyway, what I'm getting at is, that it may be good to also check whether the mail contains what we edit in the `CC` and `BCC` fields. If you feel keen to add it, perhaps add entering `BCC` and `CC` addresses to the "forwards mail successfully" test case. 

If you don't feel like messing around more with it it's fine too and we add it in a separate issue.

Let me know if you need any help with it.

Reply:
Sorry for the delay update, it should be fixed now